ViMac-Swift/Modes/HintsViewController.swift

```diff
@@ -70,22 +70,19 @@ class HintsViewController: NSViewController {
     // are you changing the location where hints are rendered?
     // make sure to update HintModeController#performHintAction as well
     func renderHint(_ hint: Hint) -> HintView? {
-        let view = HintView(associatedElement: hint.element, hintTextSize: CGFloat(textSize), hintText: hint.text, typedHintText: "")
         guard let elementFrame = self.elementFrame(hint.element) else { return nil }
         
-        let hintOrigin: NSPoint = {
-            // position hint on bottom-left of AXLinks (see #373)
-            if hint.element.role == "AXLink" {
-                return elementFrame.origin
-            }
-            
-            // position hint on center of element
-            let elementCenter = GeometryUtils.center(elementFrame)
-            return NSPoint(
-                x: elementCenter.x - (view.intrinsicContentSize.width / 2),
-                y: elementCenter.y - (view.intrinsicContentSize.height / 2)
-            )
-        }()
+        let elementCenter = GeometryUtils.center(elementFrame)
```

Comment:
the logic here is now different - hints for axlinks for now shown in the center of the link element rather than the bottom-left.

Reply:
I think to keep the implementations consistent between hint location and click location, you could use `clickPosition()`.

Reply:
Ah, that was an intentional change I didn't intend for that branch, though I do prefer it that way.  I found that my brain was having trouble parsing hints that were not in the center of the element.  I can move it to another branch if you like.